<launch>
    <node pkg="accel_control" type="position_control.py" name="position_control" output="screen">
        <param name="path" type="string" value="/home/zxh/ws/quaddif/outputs/2025-01-15/16-29-44"/>
        <param name="device" type="string" value="cpu"/>
        <param name="max_acc" type="double" value="10."/>
        <param name="max_vel" type="double" value="5."/>
        <param name="hover_thrust" type="double" value="0.707"/>
        <param name="control_freq" type="int" value="50"/>
        <!-- home and target -->
        <param name="home_x" type="double" value="0"/>
        <param name="home_y" type="double" value="0"/>
        <param name="target_x" type="double" value="10"/>
        <param name="target_y" type="double" value="-5"/>
        <param name="height" type="double" value="5"/>
    </node>
</launch>